procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -0,0 +1,631 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #get classes and start them
</pre><p>Consider removing this comment to a markdown cell and also, consider making "sections" with markdown. It makes it easier to navigate the notebook. Basically create a markdown cell above this and do:</p><p> </p><pre class="ql-syntax" spellcheck="false">## Setup observatory control classes
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>